eth/.catalog-info.yaml

```diff
@@ -18,3 +18,5 @@ spec:
   type: contract
   lifecycle: production
   system: spectre-bridge
```

Comment:
I suggest renaming it to `fast-bridge` too in the follow-up PRs (for both NEAR and Ethereum parts)